surface-model/surfacemodel_shacl.ttl

```diff
@@ -0,0 +1,137 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix surfacemodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/surfacemodel/> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+surfacemodel:surfaceModelShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surfacemodel:FormatShape ;
+            sh:order 2 ;
+            sh:path surfacemodel:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surfacemodel:DataShape ;
+            sh:order 3 ;
+            sh:path surfacemodel:data ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surfacemodel:QualityShape ;
+            sh:order 1 ;
+            sh:path surfacemodel:quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surfacemodel:QuantityShape ;
+            sh:order 0 ;
+            sh:path surfacemodel:quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 0 ;
+            sh:node surfacemodel:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path georeference:georeference ] ;
+    sh:targetClass surfacemodel:surfaceModel .
+
+surfacemodel:DataShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
```

Comment:
if there is no example then this should be removed. Or give a better example than "nan"